Makefile

```diff
@@ -534,7 +534,7 @@ generate-go-conversions-test-extension: $(CONVERSION_GEN) ## Generate conversion
 # The tmp/sigs.k8s.io/cluster-api symlink is a workaround to make this target run outside of GOPATH
 .PHONY: generate-go-openapi
 generate-go-openapi: $(OPENAPI_GEN) $(CONTROLLER_GEN) ## Generate openapi go code for runtime SDK
-	@mkdir -p ./tmp/sigs.k8s.io; ln -s $(ROOT_DIR) ./tmp/sigs.k8s.io/; cd ./tmp; \
+	@mkdir -p ./tmp/sigs.k8s.io; rm -f ./tmp/sigs.k8s.io/cluster-api; ln -s $(ROOT_DIR) ./tmp/sigs.k8s.io/cluster-api; cd ./tmp; \
```

Comment:
This is a funny fix: `rm sigs.k8s.io/cluster-api` below didn't work because CircleCI doesn't call the parent directory `cluster-api` which was assumed _here_ (it's something like `/home/circleci/project` instead).